plugin/pkg/admission/limitranger/admission.go

```diff
@@ -239,6 +239,11 @@ func mergeContainerResources(container *api.Container, defaultRequirements *api.
 	for k, v := range defaultRequirements.Limits {
 		_, found := container.Resources.Limits[k]
 		if !found {
+			request, found := container.Resources.Requests[k]
+			if found && v.Cmp(request) < 0 {
+				// skip default limit setting if container's request is larger
```

Comment:
this is a change in behavior that likely need more than just release notes.

See [comment](https://github.com/kubernetes/kubernetes/issues/112835#issuecomment-1270225840): 

> that would be a substantial breaking change, I don't think we could do that short of a v2 API and I'm not sure it would be correct behavior even then.

I'm closing this as this change is not as trivial as it sounds.

(also this PR fixes some typos, maybe we can fix them in separate PR)

/close

Reply:
It sounds correct. Thanks.

- we should document it, shouldn't we?